tutorials/rnn/ptb/ptb_word_lm.py

```diff
@@ -56,13 +59,15 @@
 from __future__ import division
 from __future__ import print_function
 
-import inspect
 import time
 
+import google3
 import numpy as np
 import tensorflow as tf
 
-import reader
+from google3.third_party.tensorflow_models.tutorials.rnn.ptb import reader
+from google3.third_party.tensorflow_models.tutorials.rnn.ptb import util
+from google3.third_party.tensorflow.python.client import device_lib
```

Comment:
Can you remove the google3 imports? Also, we don't have a `util.py` file in this folder, and I'm not sure whether we can import `device_lib`.